src/ui/button/expanded-button.module.scss

```diff
@@ -0,0 +1,71 @@
+.button {
```

Comment:
is it copy of `src/components/data-tables/columns/ConjunctionEventHistoryColumns/eventHistoryTable.module.scss` ?

We can keep one version

Reply:
yes, and i think we should have it in UI components